src/server_manager/web_app/ui_components/outline-gcp-create-server-app.ts

```diff
@@ -303,32 +307,55 @@ export class GcpCreateServerApp extends LitElement {
     } else {
```

Comment:
For readability, change to `else if` to avoid the extra nesting.

Reply:
Done